Sources/Core/Notifications/Notifications.swift

```diff
@@ -24,27 +24,26 @@ public final class Notifications: NSObject {
     /// A custom local notification content.
     public typealias LocalNotificationContent = (Message, Channel) -> UNNotificationContent
     /// A callback type to open a chat view controller with a given message reference.
-    public typealias ShowNewMessageCallback = (MessageReference) -> Void
+    public typealias NewMessageCallback = (MessageReference) -> Void
```

Comment:
```suggestion
    public typealias OnNewMessage = (MessageReference) -> Void
```
Let's be consistent. Do as we did in Client.